api/profiles/dev/application.properties

```diff
@@ -25,6 +25,7 @@ database.initial.pool.size=5
 
 #monitoring configuration
 monitoring.backend=elastic
+monitoring.stats.export.xls.template=${CP_API_MONITORING_XLS_TEMPLATE_PATH:}
```

Comment:
Let's set the environment variable in `deploy/contents/install/install-config` to some default value.

Reply:
Done